docs/source/trainer/numerics.md

```diff
@@ -0,0 +1,73 @@
+# 🔢 Numerics
+
+Composer supports several single and half-precision number formats including IEEE single-precision `fp32`, IEEE half-precision `fp16`, Google’s truncated `bf16` format and support for mixed precision training `amp`.  The following formats are supported per accelerator:
+
+| Format | CPU | GPU - A100 | GPU - V100 | GPU - T4 |
+| --- | --- | --- | --- | --- |
+| bf16 | ➖ | ✅ | ➖ | ➖ |
+| fp16 | ➖ | ✅ | ✅ | ✅ |
+| fp32 (default) | ✅ | ✅ | ✅ | ✅ |
+| amp | ➖ | ✅ | ✅ | ✅ |
+
+When using the {class}`~.Trainer`, the number format can selected by specifying the `precision` argument during initialization. In the example below, we are training on a `gpu` device and using the `fp16` format:
```

Comment:
```suggestion
When using the {class}`~.Trainer`, the number format can selected by specifying the `precision` argument during initialization. In the example below, we are training on a `gpu` device using Automatic Mixed Precision (AMP). AMP is explained below:
```

The example shows automatic mixed-precision so the line " In the example below, we are training on a `gpu` device and using the `fp16` format" is incorrect.